### PR TITLE
Document Update 

### PR DESCRIPTION
**Pull request by v-shnarang**
## Description
A note has been added to the default deployment document, specifying that the SQL migration tool should be run immediately after the deployment completes successfully, if the SQL database deployment has the Schema Update set to TOOL

## Related issues
Addresses [issue #130428](https://microsofthealth.visualstudio.com/Health/_workitems/edit/130428).

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- Tag the PR with **Schema Version backward compatible** or **Schema Version backward incompatible** or **Schema Version unchanged** if this adds or updates Sql script which is/is not backward compatible with the code.
- [ ] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: docs/DefaultDeployment.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ We recommend starting with the SQL deployment as it offers the most features. Ho
...
 
 To deploy the backend SQL Server, Azure Web App, and FHIR server code, use the buttons below to deploy through the Azure Portal. If you would like to protect the FHIR API with token authorization, you will need to supply application registration details as described above.
+###### NOTE
+>If you select a SQL database deployment with the Schema Update set to TOOL, you must run the SQL migration tool immediately after the deployment completes successfully.<br>
+The detailed information about this tool is provided [here](SchemaMigrationTool.md).
```
Comment (evachen96) [inline docs/DefaultDeployment.md:62]: Small nit-pick, can you change the second sentence to "Detailed information about this tool is provided here."

Thanks!

Comment (v-shnarang) [reply docs/DefaultDeployment.md]: @evachen96 : We have made the requested changes. Thanks !

**Review by evachen96 — APPROVED** [review]
Looks good, you can continue the process for merging the PR!

**Review by brendankowitz — APPROVED** [review]

**Pull request merged**